Userland/Applications/SoundPlayer/main.cpp

```diff
@@ -40,11 +40,6 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     auto audio_client = TRY(Audio::ConnectionToServer::try_create());
     auto decoder_client = TRY(ImageDecoderClient::Client::try_create());
 
-    Config::pledge_domain("SoundPlayer");
-    app->set_config_domain(TRY("SoundPlayer"_string));
```

Comment:
Removing these isn't a solution. We want to limit the config access as much as possible. You should be able to instead add extra config domains by calling `Config::pledge_domains({"SoundPlayer", "Whatever"});`

Reply:
Should be good now